packages/wp-source/src/libraries/handlers/date.ts

```diff
@@ -22,6 +39,13 @@ export const dateHandler: Handler = async ({
   const month = params.month && parseInt(params.month);
   const day = params.day && parseInt(params.day);
 
+  if (typeof month === "number" && (month < 1 || month > 12)) {
+    throw new ServerError(`${month} is a wrong month number`, 404);
+  }
+  if (typeof day === "number" && (day < 1 || day > 31)) {
```

Comment:
Maybe there is a way to use the real number of days for the given month an year here...

I've tested http://localhost:3000/2016/02/31/ and it's showing all posts published in March, 2016. :sweat_smile:

Reply:
fair enough :)

We can check for leap year with: https://stackoverflow.com/a/16353241/2638310